app/templates/submit.hbs

```diff
@@ -250,11 +250,53 @@
                     {{/with}}
 
                     {{! ==== COI HERE ==== }}
-                    {{!-- Add logic for COI here --}}
-                    {{!-- No: Disabled Textbox with placeholder
-                        Yes: Required textarea to describe
-                    --}}
-                    {{!-- Section explaining COI here --}}
+                    {{#with _names.[6] as |name|}}
+                        {{#preprint-form-section id="author-coi-assertion" editMode=editMode name=name allowOpen=true canEdit=canEdit errorAction=(action 'error') as |hasOpened|}}
+                            {{preprint-form-header editMode=editMode node=node name=name valid=true isValidationActive=hasOpened}}
+                            {{#preprint-form-body}}
+                                <div role='radiogroup'>
```

Comment:
I think you can use: `aria-labelledby="author-coi-assertion"` here, and also `aria-describedby="whatever-ID-the-div-at-line286-gets"` for some extra accessibility